js/utils.js

```diff
@@ -1,22 +1,40 @@
-function doloadGPTScript(resolve, reject) {
+export const loadGPTScript = () => new Promise((resolve, reject) => {
   window.googletag = window.googletag || {};
   window.googletag.cmd = window.googletag.cmd || [];
 
-  const scriptTag = document.createElement('script');
-  scriptTag.src = `${document.location.protocol}//www.googletagservices.com/tag/js/gpt.js`;
-  scriptTag.async = true;
-  scriptTag.type = 'text/javascript';
-  scriptTag.onerror = function scriptTagOnError(errs) {
-    reject(errs);
-  };
-  scriptTag.onload = function scriptTagOnLoad() {
-    resolve(window.googletag);
+  const script = document.createElement('script');
+
```

Comment:
can u please delete this extra blank lines?

Reply:
For sure!